upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+---
+title: 4.0.0
+description: Handle the changes in the43.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 4.0.0
+slug: changes-in-4-0-0
```

Comment:
Link the page from the main ("List") article.

Reply:
I will, thank you, forgot about that.